docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_2.md

```diff
@@ -1,32 +1,32 @@
+RNA-Seq data can be used to study the **transcriptome**, a collection of all the RNA transcripts from a sample. The transcriptome data can be utilized for many types of analyses. In this lesson we will focus on one such application: **Differential Gene Expression (DGE)** analysis in order to determine which genes are expressed at different levels between the conditions/groups of interest. The identified genes offer biological insight into the processes and pathways affected by the chosen experimental conditions.
 
-A typical RNAseq workflow is highlighted in the schematic diagram below. The orange boxes highlight the steps you will do in this tutorial!
+A typical RNA-Seq workflow is highlighted in the schematic diagram below. The orange boxes highlight the steps you will do in this tutorial!
 
-![RNAseq workflow](../rna-seq-images/rna-seq-workflow.jpeg "RNAseq workflow")
+![RNA-Seq workflow](./rna-seq-images/rna-seq-workflow.jpeg "RNA-Seq workflow")
 
 The first steps involve extraction, purification, and quality checks of RNA from the biological samples, followed by library preparation to convert the RNA to cDNA (complimentary DNA) fragments which are then sequenced on a high-throughput sequencer.
 
 The generated raw reads are quality checked and aligned against either a reference genome/transcriptome (if available) or used for *de novo* assembly. Based on read depth, estimates for the abundance of expression can be generated and the values can be summarized for gene-level analysis. The expression count data is used for statistical modeling and testing to identify differentially expressed genes which can be examined further via visualization and downstream functional analysis.
 
-!!! info "RNAseq Resources"
+!!! info "RNA-Seq Resources"
 
-    Learn more about RNAseq through this [video tutorial by StatQuest](https://www.youtube.com/watch?v=tlf6wYJrwKY&list=PLblh5JKOoLUJo2Q6xK4tZElbIvAACEykp&index=1). You can also follow this [end to end RNAseq workflow](https://www.bioconductor.org/packages/devel/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html) that uses well known [Bioconductor packages](http://bioconductor.org).
+    Learn more about RNA-Seq through this [video tutorial by StatQuest](https://www.youtube.com/watch?v=tlf6wYJrwKY&list=PLblh5JKOoLUJo2Q6xK4tZElbIvAACEykp&index=1). You can also follow this [end to end RNA-Seq workflow](https://www.bioconductor.org/packages/devel/workflows/vignettes/RNA-SeqGene/inst/doc/RNA-SeqGene.html) that uses well known [Bioconductor packages](http://bioconductor.org).
```

Comment:
the URL accidentally got changed too, should be: https://www.bioconductor.org/packages/devel/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html